{"uuid": "d785ebcb-569b-4102-b2f9-4aa9b42195f8", "children": ["07ab791c-6c84-48f6-8b10-cb318c637824", "bcd093ed-567a-454a-a13d-efddea39b16f", "54ec91eb-2b58-4306-a77f-dd8cfba7ebaf", "91af8d4b-3bc4-48f6-abd5-f2e6c0273f50", "8ffcd8bb-b4b0-4701-b150-a5e34c4dd619", "9907fa66-c508-4e28-8958-90c6b54c984a"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyError: 'ifname'\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 79, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_id,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 292, in setup_profiles\n    ap_wifi_data = ap_ssh.get_interface_details()\n  File \"/wlan-testing/tests/../libs/apnos/apnos.py\", line 401, in get_interface_details\n    wifi_info[j[\"ifname\"]] = [j[\"config\"][\"ssid\"], j[\"config\"][\"encryption\"], j[\"config\"][\"key\"]]\n"}, "attachments": [{"name": "ucentral_config: ", "source": "96f8b0d3-a55e-47d2-9fb1-ceb88e16b561-attachment.attach"}, {"name": "Time Took to apply Config: 34.09790015220642", "source": "756a136c-2f0d-4cd2-94ac-3889afdc40b6-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "40ea166e-2899-4adc-ba9b-809dc3448952-attachment.attach"}, {"name": "AP LOgs: ", "source": "b0674b4d-86ae-441a-915d-d0912f0c7932-attachment.attach"}], "start": 1629334697333, "stop": 1629334780798}], "start": 1629334697333, "stop": 1629334781080}